test/integration/healthController.test.js

```diff
@@ -1,12 +1,14 @@
 const chai = require('chai')
 const { expect } = chai
 const chaiHttp = require('chai-http')
+// const { config } = require('winston')
 chai.use(chaiHttp)
 
 const app = require('../../server')
 const authService = require('../../services/authService')
 const addUser = require('../utils/addUser')
-
+const config = require('config')
```

Comment:
`config` is accessible globally. Can you check if its accessible in the test scripts?
If yes, we can remove the import for it.

Reply:
Yes I tried but the tests were failing so I had to import it into all the files